website/search/elastic_search.py

```diff
@@ -294,6 +296,11 @@ def update_node(node, index=None, bulk=False):
         else:
             es.index(index=index, doc_type=category, id=elastic_document_id, body=elastic_document, refresh=True)
 
+    #TODO @hmoco, make this a celry task that takes care of updating files
+    from website.files.models.base import FileNode
+    for file_ in list(FileNode.find(Q('node', 'eq', node) & Q('provider', 'eq', 'osfstorage') & Q('is_file', 'eq', True))):
```

Comment:
As far as I know you can iterate over the queryset without converting to a list, which might be more efficient. See example above:
https://github.com/CenterForOpenScience/osf.io/pull/4358/files#diff-f12147ecc6ac2df89590531395879679R19